indico/modules/files/schemas.py

```diff
@@ -14,7 +14,7 @@
 class FileSchema(mm.ModelSchema):
     class Meta:
         model = File
-        fields = ('claimed', 'content_type', 'created_dt', 'filename', 'size', 'uuid')
+        fields = ('claimed', 'content_type', 'created_dt', 'filename', 'size', 'uuid', 'external_download_url')
```

Comment:
Including the signed URL here is dangerous!

Using `RHFileInfo` you can get the metadata of any file (using this schema) based on the UUID. And if this includes a signed download link it means as soon as you have access to any endpoint where you can upload a File, you could use Indico as a 24-hour one-click-hoster.

I would subclass `FileSchema` as `FileDownloadSchema` and add the signed download url only in there.

Reply:
Great point! TBH I can't seem to find any further needs for `external_download_url` considering it is only being used in `RHFileInfo` and `UploadFileMixin`. The attribute we need in the service is being exposed directly in `EditingRevisionUnclaimedFileSchema`, so I'm just removing it.

Reply:
I would keep the property, but maybe rename it to `signed_download_url`? Just not in a schema if you don't need it there anyway